fixtures/pages-functions-unenv-alias/functions-deps/dep.cjs

```diff
@@ -0,0 +1,9 @@
+// 2 imports required to trigger https://github.com/cloudflare/workers-sdk/issues/7572
```

Comment:
Are you sure it takes 2 aliased modules? 
I don't understand why as the banner seems to correctly be added only once?

Reply:
You're right, only 1 import triggers the issue. I've removed the unnecessary require (cross-env).

I was under the impression that the error got triggered when using at least 2 imports, but this was during the time I was trying to get the tests to run correctly at first before the fix. I was getting weird results with the tests after I started the fork.